internal/command/term/format.go

```diff
@@ -0,0 +1,18 @@
+package term
+
+import (
+	"fmt"
+	"time"
+)
+
+func BytesToMib(bytes uint64) string {
+	return fmt.Sprintf("%.3f", float64(bytes)/1024/1024)
+}
+
+func DurationToStrSeconds(duration time.Duration) string {
```

Comment:
Optional: Same thing as the one above: make a `FormatTime` function that return `1min 2sec` or something like, instead of 62.

Reply:
done, thanks!